docker-watt-image-push.sh

```diff
@@ -6,4 +6,40 @@ DIR="${BASH_SOURCE%/*}"
 watt_source_init
 docker_compose_build
 
-echo "Upload to AWS is not implemented yet. Do it manually"
+# Get the recent watt image.
+local_watt_image=`docker images | awk '{print $1}' | awk 'NR==2'`
+if [[ $local_watt_image != *watt_container ]]; then
+  echo "Could not found watt image"
+  exit 1
+fi
+echo "Image to be pushed: "$local_watt_image
```

Comment:
What about changing it to:
`echo "Image to be pushed:`
`docker images | grep $local_watt_image`
This would give more info about the image including creation date

Reply:
Done + remove tagged watt image if it exists